js/script.js

```diff
@@ -0,0 +1,118 @@
+/*= ============== SHOW MENU =============== */
+const navMenu = document.getElementById('nav-menu');
+const navToggle = document.getElementById('toggle-nav');
+const navClose = document.getElementById('close-nav');
+
+/*= ==== MENU SHOW ===== */
+/* Validate if constant exists */
+if (navToggle) {
+  navToggle.addEventListener('click', () => {
+    navMenu.classList.add('nav__show');
+  });
+}
+
+/*= ==== MENU HIDDEN ===== */
+/* Validate if constant exists */
+if (navClose) {
+  navClose.addEventListener('click', () => {
+    navMenu.classList.remove('nav__show');
+  });
+}
+
+/*= ============== REMOVE MENU MOBILE =============== */
+const navLink = document.querySelectorAll('.nav__link');
+
+function linkAction() {
+  const navMenu = document.getElementById('nav-menu');
+  // When we click on each nav__link, we remove the nav__show class
+  navMenu.classList.remove('nav__show');
+}
+
+navLink.forEach((n) => n.addEventListener('click', linkAction));
+
+const blockchainTopics = [{
+  topic: 'Blockchain',
+  description: 'Blockchain is a distributed ledger that is a public ledger of transactions. It is a system that allows users to store, share, and verify data.',
+  img: 'img/cryptocurrency.png',
+},
+{
+  topic: 'Crypto-currency',
+  description: 'Cryptocurrency is a digital asset that is generated through a process of mining.',
+  img: 'img/blockchain.png',
+},
+{
+  topic: 'Smart Contract',
+  description: 'Smart contract is a computer program that is executed on a blockchain. It is an agreement between two or more parties to perform a specific task.',
+  img: 'img/smart-contract.png',
+},
+{
+  topic: 'Dapps',
+  description: 'Dapps is a form of application that is designed to be used by both the users and the developers.',
+  img: 'img/dapps.png',
+},
+{
+  topic: 'Ethereum',
+  description: 'Ethereum is a decentralized platform that is used to create blockchains.',
+  img: 'img/ethereum.png',
+},
+];
+
+const programSection = document.getElementById('program');
+programSection.innerHTML = blockchainTopics.map((topic) => `
```

Comment:
- [ ] **Required:** Kindly fix console errors as listed in the images below.

![image](https://user-images.githubusercontent.com/27709832/162819174-e87a26f2-08f2-48dd-aea6-5f850fb97cc2.png)